packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -87,28 +88,23 @@ export class AppLauncherGitproviderService implements GitProviderService {
         if (user && user.login) {
           this.gitHubUserLogin = user.login;
           const orgs: { [name: string]: string } = {};
-          return this.getUserOrgs(user.login).pipe(
```

Comment:
Did you mean to remove this call? Will the user object always have the data we need?

Reply:
Yes, this API ```/api/services/git/user``` already return the GitHub organizations of the user. We use this API call to check if user git account is connected or not and if connected it returns all data that we need to show on the UI.